src/components/button/WorkspaceButton.vue

```diff
@@ -37,7 +42,7 @@ export default {
     border-radius: 30px;
 }
 
-.ws-settings {
+.ws-button {
```

Comment:
Is this any global class or from library? Just to be sure......

Reply:
Good point! I've moved this to a global class since we could use this again.